docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+  "name": "coastal"
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0-alpha1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0-alpha1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0-alpha1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "coastal",
+  "entries": [
+    {
+      "name": "coastal.v1.0.0-alpha1"
+    }
+  ]
+}
+EOF
+```
+
+- Build and push the FBC image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+### Create a `Catalog`
+The final preparation step is to create a `Catalog` resource that references the FBC image we built
+in the previous step and specify a polling interval of 15 seconds. This will make sure that the updates
+we make to the catalog image are reflected on our cluster relatively quickly.
+
+Use the following command to create the `Catalog` resource:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: coastal
+spec:
+  source:
+    type: image
+    image:
+      ref: quay.io/operator-framework/coastal-catalog:latest
+      pollInterval: 15s
+EOF
+```
+
+## Only install/upgrade to `v1.0.z` releases
+In this section we are going to create an `Operator` resource that attempts to install our operator
+with a version range of `1.0.x`. This version range ensures we only install z-stream releases for `v1.0`.
+
+In this case, the bundle we added to the FBC image in our preparation stepsdoesn't meet this version range 
+criteria so we expect the `Operator` installation to fail. This helps us highlight how pre-release
+versions of an operator are excluded from resolution when the version range implies that they should
+not be included.
+
+Use the following command to create the `Operator` resource:
+```sh
+kubectl apply -f - <<EOF
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: coastal
+spec:
+  packageName: coastal
+  version: 1.0.x
+EOF
+```
+
+We should see the `Operator` resource eventually has a failed resolution status. Verify this with:
+```sh
+kubectl get operator/coastal -o yaml -w
+```
+>[!NOTE]
+>The above command establishes a watch on the `Operator` resource and blocks. Once you are done verifying
+>the resolution status you can exit the command with `ctrl+c`
+
+### Update the FBC Image to contain a bundle for `v1.0.0`
+To highlight both the polling functionality and the version range constraints, lets add the `v1.0.0` bundle
+of our operator to the catalog image we created in the preparation steps and push the changes.
+
+- Overwrite the FBC YAML file to contain the previous content and the new bundle
```

Comment:
Earnest question ... would it be at all useful using some variant of [this formulary](https://olm.operatorframework.io/docs/advanced-tasks/catalog-update-formulary/) to patch the local catalog source with the new bundle, instead of overwriting with the new?

Reply:
Probably. I was going through this relatively quickly and didn't take the time to look for better resources for doing this. I'll go through and make some updates to incorporate this, thanks for the link!

Reply:
Updated in [9cecd82](https://github.com/operator-framework/operator-controller/pull/533/commits/9cecd82c89aa82af486c15b987cc97833517ffbf) to try and simplify the process a bit more than overwriting every time